Makefile

```diff
@@ -1,4 +1,8 @@
+PUBLIC_IP=$(shell ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $$1}')
```

Comment:
- hardcoding eth0 isn't portable
- I bet "inet addr:" is specific to linux ifconfig